### PR TITLE
F002 add edit remove employee information

### PR DESCRIPTION
**Pull request by Linhmyx1512**
# *SUMARY*
Add, edit and remove employee information.

# *CHANGE*
Change in some classes:
- Class QuanLy:
  - [x] Add new method: ```run()```, `xoaNhanVien()`, `suaNhanVien()`, `themNhanVien()`, `menu()`.
- Class main:
  - [x] Replace method `inThongTin()` with method `run()`.

# *CHECKLIST*
- [x] Add employee information.
- [x] Edit employee information.
- [x] Remove employee information.
- [x] Reformat code with dart format.

# *RESULT*
image

### CODE REVIEW

File: NhanVien.dart [hunk 1]
```diff
@@ -6,15 +6,17 @@ abstract class NhanVien {
...
   String diaChi;
   double luong;
-  NhanVien(
-      {required this.hoTen,
-      required this.ngaySinh,
-      required this.soDT,
-      required this.diaChi,
-      required this.luong});
+
+  NhanVien({required this.hoTen,
```
Comment (Tris1702) [inline NhanVien.dart]: Revert

File: NhanVien.dart [hunk 2]
```diff
@@ -6,15 +6,17 @@ abstract class NhanVien {
...
+    required this.ngaySinh,
+    required this.soDT,
+    required this.diaChi,
+    required this.luong});
+
   @override
   String toString() {
     return '$maNV\t$hoTen\t$ngaySinh\t$soDT\t$diaChi\t$luong';
   }
+
```
Comment (Tris1702) [inline NhanVien.dart]: Revert

File: NhanVienKinhDoanh.dart [hunk 3]
```diff
@@ -2,21 +2,22 @@ import 'NhanVien.dart';
 
 class NhanVienKinhDoanh extends NhanVien {
   int doanhSo;
-  NhanVienKinhDoanh(
```
Comment (Tris1702) [inline NhanVienKinhDoanh.dart]: Revert

File: NhanVienLapTrinh.dart [hunk 4]
```diff
@@ -2,21 +2,22 @@ import 'NhanVien.dart';
 
 class NhanVienLapTrinh extends NhanVien {
   int level;
-  NhanVienLapTrinh(
```
Comment (Tris1702) [inline NhanVienLapTrinh.dart]: Revert

File: README.md [hunk 5]
```diff
@@ -1,12 +1,18 @@
-# BTVN-Buoi-1
```
Comment (Tris1702) [inline README.md:1]: Revert

**Review by Tris1702 — COMMENTED** [review]
Revert all change is not necessary

File: NhanVien.dart (hunk 1 above)
Comment (Linhmyx1512) [reply NhanVien.dart]: I reverted.

File: QuanLy.dart [hunk 6]
```diff
@@ -1,12 +1,178 @@
...
+  List<NhanVien> dsNV = [];
+
+  QuanLy(this.dsNV);
+
+  void run() {
+    while (true) {
+      menu();
+      print('Nhap lua chon: ');
+      var choice = stdin.readLineSync();
+      while (int.tryParse(choice!) == null) {
```
Comment (Tris1702) [inline QuanLy.dart]: are you sure `choice` always `!= null` ?

File: QuanLy.dart [hunk 7]
```diff
@@ -1,12 +1,178 @@
...
     }
-}
\ No newline at end of file
+  }
+
+  void themNhanVien() {
+    print('Nhap loai nhan vien:\n' + '1: Lap trinh\n' + '2: Kinh doanh');
+
+    var loaiNV = stdin.readLineSync();
+    while (int.tryParse(loaiNV!) == null) {
```
Comment (Tris1702) [inline QuanLy.dart]: same question above

File: QuanLy.dart [hunk 8]
```diff
@@ -1,12 +1,178 @@
...
+    if (int.parse(loaiNV) == 1) {
+      print('Nhap level: ');
+      var level = stdin.readLineSync();
+      while (int.tryParse(level!) == null) {
+        print('Nhap sai, nhap lai');
+        level = stdin.readLineSync();
+      }
+
+      dsNV.add(NhanVienLapTrinh(
+          hoTen: hoTen!,
```
Comment (Tris1702) [inline QuanLy.dart]: same question above

File: QuanLy.dart (hunk 6 above)
Comment (Tris1702) [reply QuanLy.dart]: research more about null safety

**Review by Tris1702 — CHANGES_REQUESTED** [review]

**Review by Tris1702 — APPROVED** [review]

**Pull request merged**